tests/swap_test.go

```diff
@@ -37,7 +37,7 @@ import (
 
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 
-	"k8s.io/utils/pointer"
+	"k8s.io/utils/ptr"
```

Comment:
We have a simpler version with no external dependencies. See pkg/pointer.

I am not yet convince that it is worth using an external package for this simple generic need.

Reply:
it is deprecated, this is why i removed it, will look on pkg/pointer
just followed the deprecation message (which of course point to same package scope)

Reply:
you mean `"kubevirt.io/kubevirt/pkg/pointer"` ?
pkg/pointer/pointer.go
we implement ourselves there
```
func P[T any](t T) *T {
	return &t
}
```
which can be removed in favor of utils/ptr imo
```
// To returns a pointer to the given value.
func To[T any](v T) *T {
	return &v
}
```

Reply:
It is basically a one-liner. I do not think you can call it a re-implementation.
D
Depending on an external util just for this is an overkill IMO.

Reply:
used pkg/pointer